constitution.md

```diff
@@ -0,0 +1,75 @@
+# Queer Code Scotland: Constitution
+
+## Name
+
+The name of the organisation is Queer Code Scotland.
+
+## Purpose
+
+To create an inclusive and welcoming space for LGBTQ+ technologists by organising meetups in 
+Scotland.
+
+## Code of conduct
+
+Our meetups are open to anyone who would like to join. We expect all attendees to adhere to the 
+following code of conduct. https://queer-code.org/coc
+
+Anyone asked to stop unacceptable behavior is expected to stop immediately. If a community member 
+engages in unacceptable behavior, the community organisers may take any action they deem 
+appropriate, up to and including a temporary ban or permanent expulsion from the community without 
+warning (and without refund in the case of a paid event).
+
+## Membership
+
+A member is deemed as someone who is a member of our meetup group. 
+Membership is open for people to join and leave at any time. https://www.meetup.com/qcscot/members/
+
+## Subscriptions
+
+There is no cost of membership. On occasion we may decide to charge an 
+entry fee if we have to incur costs in order to put on an event, however we will endeavour to avoid 
+this wherever possible. 
+
+## Committee
+
+The committee exists primarily for the management of our finances. Other day to day activities are
+organised by the volunteers on #organisation-scotland slack channel on the queer code slack. 
+See https://queer-code.org for details on how to join this slack.
+
+The committee will be formed of a minimum of two people: a chairperson and treasurer. 
+Additional committee members may be appointed as required.
+
+Appointment to the committee will be by agreement of the existing committee members.
+
+Any committee member may step down from their position by informing the remainder of the committee.
```

Comment:
There seem to be no details on what happens if the size of the committee falls below two.

Reply:
now added :)